### ISSUE TITLE
AllTorrentSpecs: LoadFromFile: bencode: syntax error (offset: 0): unknown value type '\\x00'

### ISSUE THREAD

**Issue by shlomokraus** [ledgerwatch/erigon#10644]
#### System information

Erigon version: `2.60.1-d04796e3`

OS & Version: OSX

Commit hash: 

Erigon Command (with flags/config): ./build/bin/erigon --datadir="/Volumes/Ultra Touch/ethereum" --internalcl --torrent.download.rate=1024

Consensus Layer:

Consensus Layer Command (with flags/config):

Chain/Network: Mainnet

#### Expected behaviour
Started Erigon and let it sync. At one point I stopped it and when restarting I expect it to continue from where it stopped. 

#### Actual behaviour
Getting this: 

```
INFO[06-06|14:05:05.514] logging to file system                   log dir="/Volumes/Ultra Touch/ethereum/logs" file prefix=erigon log level=info json=false
INFO[06-06|14:05:05.514] Build info                               git_branch=release git_tag=v2.55.1 git_commit=unknown
INFO[06-06|14:05:05.514] Starting Erigon on Ethereum mainnet...
INFO[06-06|14:05:05.515] Maximum peer count                       ETH=100 total=100
INFO[06-06|14:05:05.515] starting HTTP APIs                       APIs=eth,erigon,engine
INFO[06-06|14:05:05.515] torrent verbosity                        level=WRN
INFO[06-06|14:05:07.619] Set global gas cap                       cap=50000000
INFO[06-06|14:05:07.658] [Downloader] Runnning with               ipv6-enabled=true ipv4-enabled=true download.rate=512mb upload.rate=4mb
INFO[06-06|14:05:07.658] Opening Database                         label=chaindata path="/Volumes/Ultra Touch/ethereum/chaindata"
INFO[06-06|14:05:07.663] [db] chaindata                           sizeLimit=12TB pageSize=8192
INFO[06-06|14:05:07.663] Initialised chain configuration          config="{ChainID: 1, Homestead: 1150000, DAO: 1920000, Tangerine Whistle: 2463000, Spurious Dragon: 2675000, Byzantium: 4370000, Constantinople: 7280000, Petersburg: 7280000, Istanbul: 9069000, Muir Glacier: 9200000, Berlin: 12244000, London: 12965000, Arrow Glacier: 13773000, Gray Glacier: 15050000, Terminal Total Difficulty: 58750000000000000000000, Merge Netsplit: <nil>, Shanghai: 1681338455, Cancun: <nil>, Prague: <nil>, Engine: ethash}" genesis=0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3
EROR[06-06|14:05:27.672] Erigon startup                           err="AllTorrentSpecs: LoadFromFile: bencode: syntax error (offset: 0): unknown value type '\\x00', file=/Volumes/Ultra Touch/ethereum/snapshots/._v1-000500-001000-bodies.seg.torrent"
AllTorrentSpecs: LoadFromFile: bencode: syntax error (offset: 0): unknown value type '\x00', file=/Volumes/Ultra Touch/ethereum/snapshots/._v1-000500-001000-bodies.seg.torrent
```

#### Steps to reproduce the behaviour
I deleted all the files and tried again multiple times. Whenever Erigon is restarted I continue to get this error. 
I am using an external ssd drive with exFat format. 

#### Backtrace

````
[backtrace]
````

**Comment by AskAlexSharov** [comment]
`ls -la datadir/snapshots`

**Comment by AskAlexSharov** [comment]
sorry i don't know who/why creating files which starting from `.` dot. seems it's some hidden files... i think we don't create them. 
anyway easy to fix: https://github.com/ledgerwatch/erigon/pull/10654

**Comment by yperbasis** [comment]
Fixed by PR #10654 & #10840
